truss/base/trt_llm_config.py

```diff
@@ -160,3 +146,41 @@ def requires_build(self):
     # when pydantic v2 is used here
     def to_json_dict(self, verbose=True):
         return json.loads(self.json(exclude_unset=not verbose))
+
+
+class TRTLLMSpeculativeDecodingConfiguration(BaseModel):
+    target: TRTLLMConfiguration
+    draft: TRTLLMConfiguration
+    total_token_limit: int = 500000
```

Comment:
We also need this for WiM. Should we move this out to a higher level?

Reply:
I'll opt to leave a refactor for a future PR.